omod/src/main/webapp/pages/providerDashboard.gsp

```diff
@@ -0,0 +1,85 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+%>
+<script type="text/javascript">
+    var breadcrumbs = [
+        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+        { label: "Provider DashBoard" , link: '#'}
```

Comment:
Would be great if this came from the message properties file.

Reply:
Resolved in #6